cmd/run/run.go

```diff
@@ -223,6 +223,7 @@ func MustDefaultConfigWithRandomPorts() *serverconfig.Config {
 	// Since this is used for tests, turn the following off:
 	config.Playground.Enabled = false
 	config.Metrics.Enabled = false
+	config.Experimentals = []string{}
```

Comment:
This is so that our ABAC tests still work internally?

Reply:
Correct, made it more explicit so intent is clear.